service/src/main/java/com/bucks/signly/SignlyApplication.java

```diff
@@ -0,0 +1,71 @@
+package com.bucks.signly;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+
+@SpringBootApplication
+@EnableJpaRepositories
+public class SignlyApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(SignlyApplication.class, args);
+    }
+
+    //todo: remove below and convert into tests to prove functionality ?
```

Comment:
commented code below as I fully agree with removing this whole commandLineRunner.